### ISSUE TITLE
IP and MAC info on each client

### ISSUE THREAD

**Issue by kemalbayar** [kolide/fleet#1728]
normally last two entry  show IP and MAC address of the osquery agent. We can't see IP and MAC addresses of our agents, why?

![image](https://user-images.githubusercontent.com/33925821/37660170-d499549c-2c62-11e8-8216-95c41a289846.png)

**Comment by zwass** [comment]
@kemalbayar what version of Fleet are you running? Support for windows network interfaces was added in https://github.com/kolide/fleet/pull/1700, so I'd be curious if this still repros on `master` (waiting up to an hour for the details to repopulate).

We are cutting a release soon that will include this fix and I'd love to know if it works for you.

**Comment by kemalbayar** [comment]
we're using the latest version 1.0.6

**Comment by zwass** [comment]
If you are able to test out `master` and see if it works that would be helpful.

**Comment by zwass** [comment]
I'm going to close this issue and say it's fixed by #1700. If anyone is able to test a build including that commit and finds there are still issues, please let me know.

**Comment by recvfrom** [comment]
@zwass it looks like the MAC Address and IP that get reported aren't always for the correct interface:

![image](https://user-images.githubusercontent.com/36489577/53910132-1f9d1f00-4021-11e9-91ed-92bdcd15ca8b.png)

![image](https://user-images.githubusercontent.com/36489577/53910161-317ec200-4021-11e9-965e-99eaa0cf0bd0.png)

This is using:
```
./build/fleet version --full
fleet - version 2.0.2-2-g4135ad7d
  branch: 	master
  revision: 	4135ad7d3fa654ec52375cfa1f0201b702569778
  build date: 	2019-03-05T19:18:37Z
  build user: 	mrt
  go version: 	go1.10.4
```

The logs show that it's talking to the correct IP:
```
ts=2019-03-06T20:07:09.589047498Z component=service method=GetClientConfig ip_addr=192.168.1.206:49798 err=null took=1.671134ms
```

Would it be possible to pull the IP from the connection information and then use that to select the appropriate interface and know which MAC address to display?
